docs/reference/api.rst

```diff
@@ -48,16 +51,21 @@ In order to activate the API's, you'll also need to add this to your routing:
         prefix:    /api
         resource: "@SonataMediaBundle/Resources/config/routing/api.xml"
 
+    # or for nelmio/api-doc-bundle v3
+    #sonata_api_classification:
+    #    prefix:    /api/classification
```

Comment:
I'd prefer to unalign these definitions, but if the current decision is to keep them aligned, I guess we should fix this (even in the previous routes).
```suggestion
    #    prefix:   /api/classification
```